content/docs/resiliency/tcp-keepalive.md

```diff
@@ -0,0 +1,83 @@
+---
+title: TCP keepalive
+weight: 10
+description: Manage idle and stale connections with TCP keepalive. 
+---
+
+Manage idle and stale connections with TCP keepalive.
+
+{{% callout type="warning" %}} 
+{{< reuse "docs/versions/warn-2-1-only.md" >}} {{< reuse "docs/versions/warn-experimental.md" >}}
+{{% /callout %}}
+
+## About TCP keepalive
+
+With keepalive, the kernel sends probe packets with only an acknowledgement flag (ACK) to the TCP socket of the destination after the connection was idle for a specific amount of time. This way, the connection does not have to be re-established repeatedly, which could otherwise lead to latency spikes. If the destination returns the packet with an acknowledgement flag (ACK), the connection is determined to be alive. If not, the probe can fail a certain number of times before the connection is considered stale. kgateway can then close the stale connection, which can help avoid longer timeouts and retries on broken or stale connections. 
```

Comment:
capitalize when starting a sentence
```suggestion
With keepalive, the kernel sends probe packets with only an acknowledgement flag (ACK) to the TCP socket of the destination after the connection was idle for a specific amount of time. This way, the connection does not have to be re-established repeatedly, which could otherwise lead to latency spikes. If the destination returns the packet with an acknowledgement flag (ACK), the connection is determined to be alive. If not, the probe can fail a certain number of times before the connection is considered stale. Kgateway can then close the stale connection, which can help avoid longer timeouts and retries on broken or stale connections. 
```